src/libs/ajax.js

```diff
@@ -616,18 +590,6 @@ export const Election = {
     return await res.json();
   },
 
-  findElectionReviewById: async (electionId, referenceId) => {
-    if (electionId !== undefined) {
-      const url = `${await Config.getApiUrl()}/api/electionReview/${electionId}`;
-      const res = await fetchOk(url, Config.authOpts());
-      return await res.json();
-    } else {
-      const url = `${await Config.getApiUrl()}/api/electionReview/last/${referenceId}`;
-      const res = await fetchOk(url, Config.authOpts());
-      return await res.json();
-    }
```

Comment:
This means we can remove https://github.com/DataBiosphere/consent/blob/develop/src/main/java/org/broadinstitute/consent/http/resources/ElectionReviewResource.java#L101 as well.

Reply:
I just added this deletion to the 1315 PR

Reply:
I also added the deletion of the one you commented on below getDarFinalAccessVote, to the 1315 PR as well